frontend/src/app/[locale]/saved-grants/page.tsx

```diff
@@ -17,9 +22,56 @@ export async function generateMetadata({ params }: LocalizedPageProps) {
   return meta;
 }
 
+const SavedOpportunitiesList = ({
+  opportunities,
+}: {
+  opportunities: Opportunity[];
+}) => {
+  return (
+    <ul className="usa-prose usa-list--unstyled">
+      {opportunities.map((opportunity) => (
+        <li key={opportunity?.opportunity_id}>
```

Comment:
doesn't seem like this would probably be possible, but if we could have undefined opportunities coming through that could mean non-unique keys here. We could either just assert that the opportunity will always be defined or add a unique fallback value?